services/api/service.go

```diff
@@ -421,12 +422,25 @@ func (api *RelayAPI) StartServer() (err error) {
 		}
 	}
 
-	if !foundCapellaEpoch || !foundDenebEpoch {
-		return ErrMissingForkVersions
+	// feature flags
+	ffCheckDenebForkSchedule := false // compatibility with older CL clients missing deneb fork schedule
```

Comment:
better name would be `ffRequireDenebForkSchedule` because it makes it more clear what the effect of it is